src/app-root/index.js

```diff
@@ -1,20 +1,20 @@
 import React from 'react';
 import {Redirect, Route, Switch} from 'react-router-dom';
-import ROUTES from '../routes/routes';
+import ROUTES, { LOG_IN } from '../routes/routes';
 
 const AppRoot = (props) => {
   return (
     <Switch>
       {(
         Object.keys(ROUTES).map(route => 
           <Route
-            exact
+            exact={true}
             key={route}
             path={ROUTES[route].path}
             component={ROUTES[route].component}
           />
-        ))}
-        <Redirect to={ROUTES.signIn.path} />
+      ))}
```

Comment:
Mapping over object inside of switch will cause new function call every time props change. 

This is not a problem but people will have a religious war over it. To avoid war you can pull this up to a memoized const or tell them to go talk to Dan A. and Ryan F.  :-)